tests/test_utils.py

```diff
@@ -139,15 +140,17 @@ def check_if_job_finished(job: JobDataclass):
     assert job.state == JobState.FINISHED
 
 
-def ibm_check_if_job_runner_result_correct(job: JobDataclass):
-    job.type = JobType.RUNNER
+def check_if_job_runner_result_correct(job: JobDataclass):
+    """Iterate over every result and check if the distribution of the measurement is correct"""
```

Comment:
Document assumptions about tested circuits in this method. (How they are expected to behave in Case i=0 and else.